scripts/battle/enemies/googledino.lua

```diff
@@ -57,7 +58,7 @@ function GoogleDino:onShortAct(battler, name)
         elseif battler.chara.id == "dess" then
             return "* Dess did absolutely nothing."
         elseif battler.chara.id == "brandon" then
-            return "* Brandon did something!"
+            return "* Brandon did an epic noscope!"
         elseif battler.chara.id == "dumbie" then
```

Comment:
If these were changed to just a `else` fallthrough fallback it would Look Better:tm:, though it'd be out-of-scope for this PR